doc/introduction/compiling_workflows.rst

```diff
@@ -45,17 +45,26 @@ compile your hybrid workflows:
 
     @qml.qjit
     @qml.qnode(dev)
-    def cost(params):
+    def circuit(params):
         qml.Hadamard(0)
         qml.RX(jnp.sin(params[0]) ** 2, wires=1)
         qml.CRY(params[0], wires=[0, 1])
         qml.RX(jnp.sqrt(params[1]), wires=1)
         return qml.expval(qml.PauliZ(1))
 
-The :func:`~.qjit` decorator can also be used on hybrid cost functions --
-that is, cost functions that include both QNodes and classical processing. We
-can even JIT compile the full optimization loop, for example when training
-models:
+The :func:`~.qjit` decorator can also be used on hybrid functions --
+that is, functions that include both QNodes and classical processing.
+
+.. code-block:: python
+
+    @qml.qjit
+    def hybrid_function(params, x):
+        grad = qml.grad(circuit)(params)[0]
+        return jnp.abs(grad - x) ** 2
+
+In addition, functions that are compiled with ``@jax.jit`` can contain calls
+to qjit-compiled functions. For example, below we compile the full optimization loop,
+using ``@jax.jit``, for example when training models:
```

Comment:
@josh146 line 79, please change `cost` to `circuit`?

Reply:
It's really bothering that you can add any comments to that line in GH

Reply:
done!